stats/cloud/collector.go

```diff
@@ -517,6 +522,20 @@ func (c *Collector) shouldStopSendingMetrics(err error) bool {
 	return false
 }
 
+type pushJob struct {
+	done    chan error
+	samples []*Sample
+}
+
+// ceil(a/b)
+func ceilDiv(a, b int) int {
+	r := a / b
```

Comment:
`K6_CLOUD_MAX_METRIC_SAMPLES_PER_PACKAGE=0` would panic with `divide by zero` here :)

Maybe add some config validation and extra validation here?

Reply:
it's now being checked in be259ee

Reply:
You should also check `MaxMetricSamplesPerPackage`. ;)